src/Fmt_ast.ml

```diff
@@ -2806,10 +2869,13 @@ and fmt_type_declaration c ?ext ?(pre = "") ?(brk = noop) ctx ?fmt_name
           (fmt_manifest ~priv:Public mfst $ str " =" $ fmt_private_flag priv)
         $ fmt "@ |"
     | Ptype_variant ctor_decls ->
+        let max acc d = max acc (String.length d.pcd_name.txt) in
```

Comment:
This is not counting the parenthesis of some symbol constructors:

```ocaml
type t =
  | ( :: ) of a * b
  | [] of looooooooooooooooooooooooooooooooooooooong_break
```

But it's maybe not worth the complexity.

Reply:
Fixed.